assets/rich_text_components_definitions.js

```diff
@@ -55,26 +55,6 @@ var richTextComponents = {
         "obj_type": "Filepath"
       },
       "default_value": ""
-    }, {
```

Comment:
Why are caption and alt being removed here?

Reply:
It was done by accidentally.!
Added now.
Thanks